autoswitch_virtualenv.plugin.zsh

```diff
@@ -398,7 +398,7 @@ function disable_autoswitch_virtualenv() {
 # immediately removes itself from the zsh-hook.
 # This seems important for "instant prompt" zsh themes like powerlevel10k
 function _autoswitch_startup() {
-    add-zsh-hook -D precmd _startup
+    add-zsh-hook -D precmd _autoswitch_startup
```

Comment:
Nice catch! Pity that zsh fails silently when the hook being deleted does not exist

Reply:
suppose you could check the contents of `$precmd_functions` before and/or after deleting.